source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -389,7 +374,7 @@ void BioXASAppController::createGeneralPanes()
 	QWidget* beamlineConfigurationView = createComponentView(BioXASBeamline::bioXAS());
 	addMainWindowView( beamlineConfigurationView, "Configuration", generalPaneCategeryName_, generalPaneIcon_);
 
-	beamlineStatusView_ = new CLSBeamlineStatusView(BioXASBeamline::bioXAS()->beamStatus(), false);
+	beamlineStatusView_ = new CLSBeamlineStatusView(BioXASBeamline::bioXAS(), false);
```

Comment:
It looks like the declaration of the `beamlineStatusView_` was removed from the .h file, but we still create an instance of the view here. Does this cause a crash?